### ISSUE TITLE
 Responsivize not working on puppy linux

### ISSUE THREAD

**Issue by elieserdiaz** [virejdasani/Responsivize#5]
Puppy Linux Xenial.

The attached images speak for themselves. The buttons are not shown.
![2021-08-24_083500](https://user-images.githubusercontent.com/34104884/130617484-613672eb-150b-488c-b3ad-0b467490e845.png)

Only if I set a custom responsive view on dev tools the half of the first three appears.
![Captura el 2021-08-24 a las 08 37 35](https://user-images.githubusercontent.com/34104884/130617955-7c37ac2f-d3ca-4e5d-92da-afc112edec16.png)

But your Resposivize don't look too responsive to me. And it does nothing on url input, not online or off line urls respond at all.
![2021-08-24_083915](https://user-images.githubusercontent.com/34104884/130617840-fa7c6b85-91a9-4438-b926-e64310ffe382.png)

**Comment by virejdasani** [comment]
Hi @elieserdiaz, Responsivize is not designed to be used as a website. It is an app. Please run the executable file. See [here](https://docs.appimage.org/introduction/quickstart.html) on how to execute files :)

**Comment by elieserdiaz** [comment]
Could you please show me where exactly is the resposivisize.appimage file that i can not see it?
![2021-08-24_113605](https://user-images.githubusercontent.com/34104884/130646863-3e3131c1-8a11-4a6c-a62f-39569c532476.png)

**Comment by virejdasani** [comment]
It's in the bottom right corner. It is the first one in the third row (rows are horizontal)

**Comment by elieserdiaz** [comment]
> It's in the bottom right corner. It is the first one in the third row (rows are horizontal)

Yes, I noticed but it throws me an error when call it through the command line. I did this because it did nothing when I clicked on it. Should it works if I create a symlink on usr/bin/ pointing to this file?

**Comment by virejdasani** [comment]
Have you checked the box in the app properties that says `allow executing..`?

**Comment by steffenu** [comment]
Im on unbuntu. Checkbox is ticked. But im getting "is not an application error"
